ci/azure-pipelines-docs-integration.yml

```diff
@@ -342,6 +342,42 @@ stages:
             AWS_SECRET_ACCESS_KEY: $(AWS_SECRET_ACCESS_KEY)
             AWS_DEFAULT_REGION: $(AWS_DEFAULT_REGION)
 
+    - job: test_docs_aws_spark
+      timeoutInMinutes: 30
+      condition: or(eq(stageDependencies.scope_check.changes.outputs['CheckDocsDependenciesChanges.DocsDependenciesChanged'], true), eq(stageDependencies.scope_check.changes.outputs['CheckDocsChanges.DocsChanged'], true), eq(variables.isDevelop, true), eq(variables.isManual, true))
+      variables:
+        python.version: '3.8'
+
+      steps:
+        - task: UsePythonVersion@0
+          inputs:
+            versionSpec: '$(python.version)'
+          displayName: 'Use Python $(python.version)'
+        - script: |
+            pip install pyspark==3.3.2
+          displayName: Install Pyspark 3.3.2
+        - script: |
+            export JAVA_HOME=$JAVA_HOME_11_X64
+          displayName: Set JAVA_HOME  to existing directory
+        - script: |
+            wget https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-aws/3.3.2/hadoop-aws-3.3.2.jar
+            wget https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-bundle/1.11.1026/aws-java-sdk-bundle-1.11.1026.jar
+            export pyspark_dir=`python -c 'import pyspark; print(pyspark.__path__[0] + "/jars/")'`
+            mv hadoop-aws-3.3.2.jar $pyspark_dir
+            mv aws-java-sdk-bundle-1.11.1026.jar $pyspark_dir
```

Comment:
Non blocking suggestion so we don't forget to update one of the values in the future (please double check my syntax though if you decide to incorporate!):

```suggestion
      variables:
        python.version: '3.8'
        spark.version: '3.3.2'
        matching_aws_java_sdk_bundle_version: '1.11.1026'

      steps:
        - task: UsePythonVersion@0
          inputs:
            versionSpec: '$(python.version)'
          displayName: 'Use Python $(python.version)'
        - script: |
            pip install pyspark==$(spark.version)
          displayName: Install Pyspark $(spark.version)
        - script: |
            export JAVA_HOME=$JAVA_HOME_11_X64
          displayName: Set JAVA_HOME  to existing directory
        - script: |
            wget https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-aws/$(spark.version)/hadoop-aws-$(spark.version).jar
            wget https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-bundle/$(matching_aws_java_sdk_bundle_version)/aws-java-sdk-bundle-$(matching_aws_java_sdk_bundle_version).jar
            export pyspark_dir=`python -c 'import pyspark; print(pyspark.__path__[0] + "/jars/")'`
            mv hadoop-aws-$(spark.version).jar $pyspark_dir
            mv aws-java-sdk-bundle-$(matching_aws_java_sdk_bundle_version).jar $pyspark_dir
```

Reply:
This is a great idea @anthonyburdi. Thank you